6-Final-Proj/client/src/components/TodoItem/TodoItem.driver.tsx

```diff
@@ -0,0 +1,49 @@
+import { TodoItem } from "./TodoItem";
+import { render, fireEvent } from "@testing-library/react";
+import { TodoItem as TodoItemData } from "../../../../common/types";
+import { aRandomItem } from "../../test/utils"; 
+import { BaseDriver } from "../../test/baseDriver";
+import hooks from "../../dataHooks";
+
+export class TodoItemDriver extends BaseDriver {
+    private removeCallback: jest.Mock = jest.fn();
+    private updateCallback: jest.Mock = jest.fn();
+    private todo: TodoItemData = aRandomItem();
+
+    private editBtn = () => (this.withWrapper(hooks.editBtn));
+    private removeBtn = () => (this.withWrapper(hooks.removeBtn));
+    private checkBtn = () => (this.withWrapper(hooks.checkBtn));
+    private confirmBtn = () => (this.withWrapper(hooks.confirmBtn));
+    private label = () => (this.withWrapper(hooks.label));
+    private input = () => (this.withWrapper(hooks.input));
```

Comment:
maybe this members shouldn't be a function

Reply:
why? I can reuse this any time I'd like to access the components from when/get

Reply:
you're right